base/cvd/cuttlefish/host/commands/cvd/parser/cf_flags_validator.cpp

```diff
@@ -38,120 +39,15 @@ using Json::ValueType::objectValue;
-          {"serial_number", ConfigNode{.type = stringValue}},
-          {"use_random_serial", ConfigNode{.type = stringValue}},
-          {"guest_enforce_security", ConfigNode{.type = booleanValue}},
-        }}},
-        {"disk", ConfigNode{.type = objectValue, .children = {
-          {"default_build", ConfigNode{.type = stringValue}},
-          {"super", ConfigNode{.type = objectValue, .children = {
-            {"system", ConfigNode{.type = stringValue}},
-          }}},
-          {"download_img_zip", ConfigNode{.type = booleanValue}},
-          {"download_target_zip_files", ConfigNode{.type = booleanValue}},
-          {"blank_data_image_mb", ConfigNode{.type = uintValue}},
-          {"otatools", ConfigNode{.type = stringValue}},
-        }}},
-        {"graphics", ConfigNode{.type = objectValue, .children = {
-          {"displays", ConfigNode{.type = arrayValue, .children = {
-            {kArrayValidationSentinel, ConfigNode{.type = objectValue, .children {
-              {"width", ConfigNode{.type = uintValue}},
-              {"height", ConfigNode{.type = uintValue}},
-              {"dpi", ConfigNode{.type = uintValue}},
-              {"refresh_rate_hertz", ConfigNode{.type = uintValue}},
-            }}},
-          }}},
-          {"record_screen", ConfigNode{.type = booleanValue}},
-        }}},
-        {"streaming", ConfigNode{.type = objectValue, .children = {
-          {"device_id", ConfigNode{.type = stringValue}},
-        }}},
-        {"connectivity", ConfigNode{.type = objectValue, .children = {
-          {"vsock", ConfigNode{.type = objectValue, .children = {
-            {"guest_group", ConfigNode{.type = stringValue}},
-          }}}
-        }}}
-      }}},
-    }}},
-  {"fetch", ConfigNode{.type = objectValue, .children = {
-      {"api_key", ConfigNode{.type = stringValue}},
-      {"credential_source", ConfigNode{.type = stringValue}},
-      {"wait_retry_period", ConfigNode{.type = uintValue}},
-      {"external_dns_resolver", ConfigNode{.type = booleanValue}},
-      {"keep_downloaded_archives", ConfigNode{.type = booleanValue}},
-      {"api_base_url", ConfigNode{.type = stringValue}},
-    }}},
-  {"metrics", ConfigNode{.type = objectValue, .children = {
-      {"enable", ConfigNode{.type = booleanValue}},
-    }}},
-  {"common", ConfigNode{.type = objectValue, .children = {
-    {"group_name", ConfigNode{.type = stringValue}},
-    {"host_package", ConfigNode{.type = stringValue}},
-  }}},
-},
-};
-
 }  // namespace
 
 Result<void> ValidateCfConfigs(const Json::Value& root) {
   static const auto& kSupportedImportValues =
       *new std::unordered_set<std::string>{"phone", "tablet", "tv", "wearable",
                                            "auto",  "slim",   "go", "foldable"};
 
-  CF_EXPECT(Validate(root, kRoot), "Validation failure in [root object] ->");
+  cvd::config::Launch launch_config;
+  CF_EXPECT(Validate(root, launch_config), "Validation failure in [root object] ->");
```

Comment:
NIT: we should probably change the format of this, it was meant to visually build up from the previous `Validate` implementation `CF_EXPECT` errors